sdk/include/opentelemetry/sdk/metrics/measurement_processor.h

```diff
@@ -0,0 +1,108 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+#ifndef ENABLE_METRICS_PREVIEW
+
+#  include "opentelemetry/common/key_value_iterable_view.h"
+#  include "opentelemetry/sdk/metrics/instruments.h"
+#  include "opentelemetry/sdk/metrics/metric_reader.h"
+#  include "opentelemetry/sdk/metrics/state/sync_metric_storage.h"
+
+#  include <map>
+
+OPENTELEMETRY_BEGIN_NAMESPACE
+namespace sdk
+{
+namespace metrics
+{
+
+static std::size_t MakeKey(const MetricReader &metric_reader)
```

Comment:
Do we need this function introduced to the namespace?

Reply:
Wanted to avoid a global static function in the default namespace, and since it calculates hash on MetricReader pointer, thought it is better to make it part of opentelemetry::metrics namespace. Let me know if it doesn't look correct?